packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx

```diff
@@ -145,26 +142,22 @@ const ViewerHome: React.FC = () => {
         onIModelAppInit={onIModelAppInit}
         uiProviders={[
           new ViewerNavigationToolsProvider(),
-          new ViewerContentToolsProvider({
-            vertical: {
-              measureGroup: false,
-            },
-          }),
+          new ViewerContentToolsProvider(),
```

Comment:
rever the changes you made here

Reply:
changed

Reply:
im not seeing it...

Reply:
![image](https://user-images.githubusercontent.com/11051042/220453466-798c6ce5-859f-456b-913f-b829b2f9f0ef.png)